README.md

```diff
@@ -26,28 +33,122 @@ There is a large number of templates that are ready to use and are officially su
+| `@asyncapi/python-paho-template`              | Generates Python service that uses Paho library                 | [click here](https://github.com/asyncapi/python-paho-template)              |
+| `@asyncapi/html-template`                     | Generates HTML documentation site                               | [click here](https://github.com/asyncapi/html-template)                     |
+| `@asyncapi/markdown-template`                 | Generates documentation in Markdown file                        | [click here](https://github.com/asyncapi/markdown-template)                 |
+| `@asyncapi/ts-nats-template`                  | Generates TypeScript NATS client                                | [click here](https://github.com/asyncapi/ts-nats-template/)                 |
+| `@asyncapi/go-watermill-template`             | Generates Go client using Watermill                             | [click here](https://github.com/asyncapi/go-watermill-template)             |
+| `@asyncapi/dotnet-nats-template`              | Generates .NET C# client using NATS                             | [click here](https://github.com/asyncapi/dotnet-nats-template)              |
+| `@asyncapi/php-template`                      | Generates PHP client using RabbitMQ                             | [click here](https://github.com/asyncapi/php-template)                      |
+| `@asyncapi/dotnet-rabbitmq-template`          | Generates .NET C# client using RabbitMQ                         | [click here](https://github.com/asyncapi/dotnet-rabbitmq-template)          |
 
 <!-- TEMPLATES-LIST:END -->
 
 You can find above templates and the ones provided by the community in **[this list](https://github.com/search?q=topic%3Aasyncapi+topic%3Agenerator+topic%3Atemplate)**
 
+# Generator Filters
+
+This library contains generator filters that can be reused across multiple templates, helping to avoid redundant work. These filters are designed specifically for Nunjucks templates and are included by default with the generator, so there's no need to add then to  dependencies seprately.
+
+This library consists of:
+
+- Custom filters. Check out [API docs](apps/nunjucks-filters/docs/api.md) for complete list
+- Lodash-powered filters. For the list of all available filters check [official docs](https://lodash.com/docs/)
+
 ## Contributing
 
+To Add a major/minor/patch Type Changes:
+
+### Manual
+1.  Create a new markdown file in the `.changeset` directory. The filename should be descriptive of the change.
+2.  Add the following content to the file in this particular format:
+
+```markdown
+---
+"@package-name-1": [type] (major/minor/patch)
+"@package-name-2": [type]
+---
+
+[Provide a brief description of the changes. For example: Added a new Release GitHub Flow to the Turborepo. No new features or bugfixes were introduced.]
+```
```

Comment:
```suggestion
    ```markdown
    ---
    "@package-name-1": [type] (major/minor/patch)
    "@package-name-2": [type]
    ---

    [Provide a brief description of the changes. For example: Added a new Release GitHub Flow to the Turborepo. No new features or bugfixes were introduced.]
    ```
```

Reply:
After the format example, I would add something like the following to better explain each part of the markdown file:

Inside the file, list the affected packages by their names, such as @package-name-1, followed by the type of change, which can be `major`, `minor`, or `patch`. The type should be specified in parentheses. For example, if you made a non-breaking feature addition to the `@asyncapi/generator` package, you would list it as `"@asyncapi/generator": minor`. 

After listing all relevant packages and their change types, provide a brief description of the changes made. This description should include a clear summary of what was added, fixed, or improved, and why these changes were necessary.